devops/templates/quiz.html

```diff
@@ -20,10 +23,13 @@
                 {% csrf_token %}
                     <ol class="nested">
                         {% for q in questions %}
-                            <li>
-                                {{ q.text }}
-                            </li>
-                            <ol class="nested" type='a'>
+
+                             <div class="card" style="margin-top:2vh;font-size:1.5rem;">
+                              <div class="card-body">
+                                <h5 class="card-title" style="font-size:1.5rem;"> {{ q.text }}</h5>
+
+                                <p class="card-text">
```

Comment:
There are extra "card" boxes between the questions and choices and also after the choices. Can you fix those?